etna/core/tests/test_validators.py

```diff
@@ -0,0 +1,62 @@
+from django.core.exceptions import ValidationError
+from django.test import SimpleTestCase
+
+from etna.core.fields import DateInputField
+
+
+class DateFieldValidatorTest(SimpleTestCase):
```

Comment:
From the name of the module, I would expect this file to contain unit tests for the validators themselves, but we are testing the field again here. I would just move this test case to `test_fields.py` and remove this module.

Reply:
As mentioned in the doc string, it validates the overridden default messages. I've split the test case for each field validator.